brain-marks/Tweets/Views/TweetCard.swift

```diff
@@ -40,14 +40,37 @@ struct TweetHeaderView: View {
 }
 
 struct TweetBodyView: View {
-    let tweetBody: String
+    @State var tweetBody: String
     var body: some View {
-        Text(tweetBody)
+//        hilightedText(str: tweetBody, searched: tweetBody.components(separatedBy: " ").first(where: { str in
+//            str.contains("#")
+//        }) ?? "#nil")
+        hilightedText(str: tweetBody)
+            .multilineTextAlignment(.leading)
             .font(.body)
             .lineSpacing(8.0)
             .padding(EdgeInsets(top: 0, leading: 18, bottom: 18, trailing: 18))
             .fixedSize(horizontal: false, vertical: true)
     }
+    
+    func hilightedText(str: String) -> Text {
+        var result = Text("")
+        let parts = str.components(separatedBy: " ")
+        print("parts: \(parts)")
+        for part in parts {
+            if part.contains("#") {
+                
+                let newResult = result + Text(" \(part)")
```

Comment:
`result` doesn't actually need to be `Text`, you can use a `String` and you could use `+=` rather than needing the `newResult`

At the end before the return you can make a Text object

Reply:
But it needs to be blue and bold

Reply:
You can do this instead: `result = result + Text("\(part)")`

Reply:
However, SwiftLint then gives me an error

Reply:
Can you show a screenshot? I didn't get an error on my side.
Also I added a change, be sure to update your branch with the changes from dev